### ISSUE TITLE
(deeplabV3 plus)  the function of get_branch_logits ?

### ISSUE THREAD

**Issue by DehuaTang** [tensorflow/models#5281]
1.System information

the top-level directory of the model **: /home/yswang/models3/research/deeplab
OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04
TensorFlow installed from (source or binary): pip
TensorFlow version (use command below): tensorflow-gpu 1.8.0
Bazel version (if compiling from source): None.
CUDA/cuDNN version: 9 7.
GPU model and memory: Titan xp / 24GB / 2 GPU
Exact command to reproduce:
for the first 30k iterations :
NUM_ITERATIONS=30000
CUDA_VISIBLE_DEVICES=2,3 python "${WORK_DIR}"/train.py
--logtostderr
--train_split="trainaug"
--model_variant="mobilenet_v2"
--atrous_rates=6
--atrous_rates=12
--atrous_rates=18
--output_stride=16
--decoder_output_stride=4
--train_crop_size=512
--train_crop_size=512
--num_clones=2
--train_batch_size=32
--dataset="pascal_voc_seg"
--training_number_of_steps="${NUM_ITERATIONS}"
--fine_tune_batch_norm=True
--tf_initial_checkpoint="${INIT_FOLDER}/mobilenet_v2/mobilenet_v2_1.0_224.ckpt"
--train_logdir="${TRAIN_LOGDIR}/gpu_test/20"
--dataset_dir="${PASCAL_DATASET}"

Describe the problem
I trained the deeplabv3+ models with the mobilenet_v2_1.0_224 versions.
I noticed the code in **model.py**, the function **def get_branch_logits**  used **so many 1*1 conv2d**
to form the final logits . 
            
        branch_logits.append(
            slim.conv2d(
                features,
                num_classes,
                kernel_size=kernel_size,
                rate=rate,
                activation_fn=None,
                normalizer_fn=None,
                scope=scope))

      return tf.add_n(branch_logits)

Can you explain it to me?
I think **only one 1*1 conv2d** can form the final logits . 
**In my experiment，using only one 1*1 conv2d can get better performance than the code.**
How about you ? Is there any difference for the performance in your experiment ? 
Waiting for your replay !

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code

**Comment by DehuaTang** [comment]
Thank you for your reply !
I notice you have written custom code. 
But I used ASPP and decoder, so the code show we must use as many as the number of ASPP  1*1 conv2d to  form the final logits . 
I can't find any relationship between the ASPP and the number of 1*1 conv2d .
How do you think about it ?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
